### ISSUE TITLE
Cache and CSRF tokens

### ISSUE THREAD

**Issue by bleepsandblops** [enupal/stripe#145]
### Description
Hello,

Am testing the plugin using Blitz, and weirdly it seems to work fine despite the CSRF tokens not changing. 

I'm using a basic form with no fields, just the button to go to stripe checkout.

I've opened two different windows in two different browsers, they have the same CSRF_TOKEN in the source code (as expected, because of Blitz cache), but everything seems to work fine in test mode, my orders go through and they are saved in the back-end.

Shouldn't they fail? Or is it because there is no custom form fields that it works?

### Steps to reproduce

1.
2.

### Additional info

- Craft version: 3.5.14
- PHP version: 7.4
- Database driver & version: Mysql 8
- Plugin version: 3.15
- Is [SCA](https://docs.enupal.com/stripe-payments/getting-started/sca.html) and Stripe Checkout enabled?: Yes

**Comment by bleepsandblops** [comment]
So I assume it's because the controller 
https://github.com/enupal/stripe/blob/53dcbf0bb9091f420a80e120142edf411837bf29/src/controllers/CheckoutController.php
is extending FrontEndController
which has 
````
// Disable CSRF validation for the entire controller
    public $enableCsrfValidation = false;
````

**Comment by andrelopez** [comment]
Hi @bleepsandblops I was able to recreate this issue and will be fixed soon

**Comment by andrelopez** [comment]
Hi @bleepsandblops we just released Stripe Payments v3.2.1 with a fix to this issue. Thanks for reporting
